packages/application-components/src/components/modal-pages/custom-form-modal-page/README.md

```diff
@@ -0,0 +1,76 @@
+| `formControls`            | `node`             |    -     | -                        | Pass a React.node with form controls. You can use the pre-made form buttons exposed by this component or you can use your own.                                                                                                                                                                                                                                                                                                                                                                                      |
+| `getParentSelector`       | `function`         |    -     | -                        | The function should return an HTML element that will be used as the parent container to hold the modal DOM tree. If no function is provided, it's expected that an HTML element with the `id="parent-container"` is present in the DOM. In `NODE_ENV=test` environment, the default HTML element is `body`.                                                                                                                                                                                                         |
+| `hideControls`            | `bool`             |    -     | `false`                  | If truthy, hides the form controls.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `shouldDelayOnClose`      | `bool`             |    -     | `true`                   | Sets whether the ModalPage should delay calling its `onClose` function to allow the closing animation time to finish. This can be turned off if the developer is controlling the ModalPage only through the `isOpen` prop, and not abruptly mounting/unmounting it or one of its parent elements. You might also want to turn this off if you need to display a Prompt (e.g. to save changes) on the ModalPage before navigating out of it, as this option makes the Modal close itself before `onClose` is called. |
+
+## Static properties
+
+### Form Control Buttons
+
+Pre-made form control buttons which can be useful for the most common use cases: Confirm, Cancel, and Delete
+
+```js
+CustomFormModalPage.FormPrimaryButton = FormPrimaryButton;
+CustomFormModalPage.FormSecondaryButton = FormSecondaryButton;
+CustomFormModalPage.FormDeleteButton = FormDeleteButton;
+```
+
+### `CustomFormModalPage.Intl`
```

Comment:
PS: I would recommend to point to the shared messages that are exposed from the `i18n` package

Reply:
I think it's the shared messages underneath. I think it's nice consumer ergonomics to expose them on the component so he doesn't have o install another package and import it.

Reply:
Yes, it's the shared messages underneath. Should we mention that here in the Readme?

Reply:
I wouldn't. I don't think it should matter to the consumer how we compose below ourselves :)

Reply:
Actually I would prefer for people to use the shared messages directly. For example, it's confusing if you need to provide a message that is not exposed by the component. I would also avoid unnecessary static exports.

I would actually remove those exports from the other components as well in the v16 release.

Reply:
Okay. Let's keep it like this here cause it's consistent with other PRs and not to dive into discussion land again :). Then we can spin up a PR to remove them. Even though I feel I disagree with doing that but at least it's a seperate discussion then.